src/components/ebay-combobox/examples/05-show-all-options/template.marko

```diff
@@ -0,0 +1,7 @@
+<ebay-combobox name="example2text" value="Basic Offer" autocomplete="list" expanded>
```

Comment:
Shouldn't `autocomplete` be set to "none"?

Reply:
...which essentially just turns this into a default combobox.

Reply:
But then that would disable filtering of the options when they type.

Reply:
So I'm confused as to the purpose of this new example. It's name "show all options", but if autocomplete is set to list then we should only show the last of those 3 options.

Reply:
`expanded` only affects the initial state of the listbox (all options) when it the expander expands it. But, while it's expanded it can still be filtered when a person begins typing.

That is in contrast to when `expanded=false`, and the expander expands the listbox, it shows a filtered set of results.

That's the only change: the initial list of the listbox options. `expander` doesn't affect filtering, typing, keyboard, etc. If we were to set `autocomplete="none"`, we would lose filtering.

Reply:
Therefore, that's what this example shows.

Reply:
Expanded and filtered should be completely independent of each other.

Reply:
To be clear, this example shows what to me is an invalid state. While autocomplete is set to list, filtering should be on. If we want to show all options while combobox has an initial expanded state (not sure why this would ever be the case, comboboxes are always collapsed until input receives focus), autocomplete should be set to "none".